README.md

```diff
@@ -2,7 +2,7 @@
 ## Example
 ``` hcl
 module external_dns {
-  source       = "git::https://github.com/provectus/swiss-army-kube.git//modules/system/external-dns"
+  source       = "git::https://github.com/provectus/sak-external-dns"
```

Comment:
terraform can use the Github repo without scheme and protocol. Like that "github.com/provectus/sak-external-dns"

Reply:
Ok